modules/navier_stokes/doc/content/source/fvbcs/PINSFVStrongBC.md

```diff
@@ -0,0 +1,24 @@
+# PINSFVStrongBC
+
+!syntax description /FVBCs/PINSFVStrongBC
+
+## Overview
+
+This object is very similar to [PCNSFVStrongBC.md]. However, this object is
+designed for use in tandem with an incompressible/weakly-compressible set of
+objects (see [PINSFVMomentumAdvection.md]) while [PCNSFVStrongBC.md] is meant
+for use in tandem with a fully compressible set of objects (see [PCNSFVKT.md]).
+
+This object accepts functions describing boundary values for pressure and
+superficial velocity. If no function is provided for a quantity, the boundary
+value of that quantity will be determined by extrapolating from the neighboring
+boundary cell centroid using cell centroid value and gradient information. From
+this mix of user-provided functions and extrapolated boundary values for
+pressure and superficial velocity, the fluxes for mass and momentum can be
+computed.
+
```

Comment:
need to stay that should use PINSFVMomentumPressureFlux with it since it assumes an IBP

Reply:
Added this
```
Note that if this object is used in conjunction with
[PINSFVMomentumPressureFlux.md], then `force_boundary_execution` must be set to
`false` in the momentum pressure flux input file block since `PINSFVFunctorBC`
already adds the pressure flux contribution no the boundary.
```